NitroxClient/Communication/Packets/Processors/GrapplingHookMovementProcessor.cs

```diff
@@ -0,0 +1,45 @@
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.MonoBehaviours;
+using NitroxClient.Unity.Helper;
+using NitroxModel_Subnautica.DataStructures;
+using NitroxModel_Subnautica.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors;
+
+public class GrapplingHookMovementProcessor : ClientPacketProcessor<GrapplingHookMovement>
+{
+    public override void Process(GrapplingHookMovement packet)
+    {
+        Exosuit exosuit = NitroxEntity.RequireObjectFrom(packet.ExosuitId).RequireComponent<Exosuit>();
+
+        IExosuitArm arm = null;
+
+        switch (packet.ArmSide)
+        {
+            case Exosuit.Arm.Left:
+                arm = exosuit.leftArm;
+                break;
+            case Exosuit.Arm.Right:
+                arm = exosuit.rightArm;
+                break;
+        }
```

Comment:
Would use `IExosuitArm arm = packet.ArmSide == Exosuit.Arm.Left ? exosuit.leftArm : exosuit.rightArm;` like in `ExosuitArmActionProcessor`